sqlcapture/discovery_test.go

```diff
@@ -16,52 +16,52 @@ func TestRecommendedCatalogName(t *testing.T) {
 		{
 			schema: "something",
 			table:  "typical",
-			want:   "something.typical",
```

Comment:
The changes in this file aren't from this PR, but I found that they needed updated when running all of the `sqlcapture` tests.